resources/js/store/modules/user/actions.js

```diff
@@ -44,8 +44,6 @@ export const actions = {
         .then(response => {
             console.log('send_password_reset_link', response);
 
-            if (!response.data.success) return;
```

Comment:
I don't understand this, really. What more do you want to do after the user requests a link, other than showing them that we've emailed their link?
The current implementation returns a redundant 'success' attribute since there are only two possible outputs from the API, either successful or with errors. If we have a successful response, we'll go to the `then` block, and the 'success' parameter is always true in this case, from the backend. That's why this condition is unnecessary, you can't enter it whatever happens. If we have an error, we automatically enter the `error` block below, which handles the error and shows it to the user.

Reply:
Sometimes its usual to return false from a controller. I find the success return method a good strategy to always show the intended success notification when you have a success response.  We keep it for now

Reply:
We also need to start handling false responses